src/applications/disability-benefits/all-claims/tests/components/ComboBox.unit.spec.jsx

```diff
@@ -36,111 +40,375 @@ describe('ComboBox', () => {
     };
   });
 
-  describe('Rendering', () => {
-    it('should render listbox with no items by default', () => {
+  describe('Default Rendering', () => {
+    it('should render with label and input', () => {
+      const { getByTestId } = render(<ComboBox {...props} />);
+
+      const input = getByTestId('combobox-input');
+
+      expect(input).to.have.attribute('label', 'Test label');
+      expect(input).to.be.visible;
+    });
+
+    it('should render required', () => {
+      const { getByTestId } = render(<ComboBox {...props} />);
+
+      const input = getByTestId('combobox-input');
+
+      expect(input).to.have.attribute('required');
+    });
+
+    it('should render with no value', () => {
+      const { getByTestId } = render(<ComboBox {...props} />);
+
+      const input = getByTestId('combobox-input');
+
+      expect(input).to.have.value('');
+    });
+
+    it('should render with listbox with no items', () => {
```

Comment:
Going back to our earlier conversation about why the listbox is even rendered when it's not visible at all... after playing around with the code, I think that it was like that in order to manage focus. So that you're not trying to move focus into the listbox when it's not even on the page yet. You can tell me when we do the refactor, but I think we'll probably stick with this choice. For now I'm handling [my issue ](https://github.com/department-of-veterans-affairs/vagov-claim-classification/issues/570) by conditionally hiding the described by text & including aria-hidden.